ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "${BASE_DOMAIN:?BASE_DOMAIN env variable should be defined}" > "${SHARED_DIR}"/basedomain.txt
+
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
+base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
+cluster_domain="${cluster_name}.${base_domain}"
+
+export AWS_DEFAULT_REGION=us-west-2  # TODO: Derive this?
+export AWS_SHARED_CREDENTIALS_FILE=/var/run/vault/nutanix/.awscred
+export AWS_MAX_ATTEMPTS=50
+export AWS_RETRY_MODE=adaptive
+export HOME=/tmp
+
+if ! command -v aws &> /dev/null
+then
+    echo "$(date -u --rfc-3339=seconds) - Install AWS cli..."
+    export PATH="${HOME}/.local/bin:${PATH}"
+    if command -v pip3 &> /dev/null
+    then
+        pip3 install --user awscli
+    else
+        if [ "$(python -c 'import sys;print(sys.version_info.major)')" -eq 2 ]
+        then
+          easy_install --user 'pip<21'
+          pip install --user awscli
+        else
+          echo "$(date -u --rfc-3339=seconds) - No pip available exiting..."
+          exit 1
+        fi
+    fi
+fi
+
+source "${SHARED_DIR}/nutanix_context.sh"
+
+hosted_zone_id="$(aws route53 list-hosted-zones-by-name \
+            --dns-name "${base_domain}" \
+            --query "HostedZones[? Config.PrivateZone != \`true\` && Name == \`${base_domain}.\`].Id" \
+            --output text)"
+echo "${hosted_zone_id}" > "${SHARED_DIR}/hosted-zone.txt"
+
+echo "$(date -u --rfc-3339=seconds) - Creating DNS records ..."
+cat > "${SHARED_DIR}"/dns-create.json <<EOF
+{
+"Comment": "Create OpenShift DNS records for Nutanix IPI CI install",
+"Changes": [{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api.${cluster_domain}.",
+      "Type": "A",
+      "TTL": 60,
+      "ResourceRecords": [{"Value": "${API_VIP}"}]
+      }
+    },{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api-int.${cluster_domain}.",
```

Comment:
api-int should really not be here

This was added to vSphere for the windows folks but only as a workaround. The api-int A record should be supplied by coredns static pod

Reply:
We've checked the install process without the record. Master nodes even can not successfully start.
Looks like we need the workaround as well like vSphere
<img width="729" alt="Screenshot 2022-04-14 at 17 12 43" src="https://user-images.githubusercontent.com/506414/163421107-644635a7-77ca-46a4-bc17-011184ef17d1.png">
.

Reply:
That is a bug then

see:
https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/node.go#L32-L44

Reply:
Thank you. We'll remove it.